tests/test_core/test_docker_compose.py

```diff
@@ -64,8 +64,7 @@ def test_can_parse_multiple_compose_files():
         assert host == "0.0.0.0"
         assert port == "3306"
 
-        host = compose.get_service_host("hub", 4444)
```

Comment:
One more small change: let's check both methods for getting the information here to ensure we main backwards compatibility.